sql/ddl/create/primary-key.md

```diff
@@ -26,22 +26,20 @@ tags:
 
 links:
 
-
+  - '[Column Constraints](https://en.wikibooks.org/wiki/Structured_Query_Language/Create_Table#Column_Constraint){website}'
 
 aspects:
   - introduction
   - workout
 
 ---
 
-# Primary and Foreign Keys
+# Primary Key
 
 ---
 ## Content
 
-Due to the nature of Relational Databases, rows should be identifiable without difficulty. When defining a new table, there are two types of constraints that can be declared: `PRIMARY KEY`s and `FOREIGN KEY`s.
-
-### Primary Key
+Due to the nature of Relational Databases, rows should be identifiable without difficulty. When defining a new table, there are two types of constraints that can be declared, `PRIMARY KEY`s and `FOREIGN KEY`s, and this insight will focus on the former. 
```

Comment:
I think this sentence should explain a bit why easy identification matters:

> Due to the inherent nature of Relational Databases, rows should be easily identifiable because <todo>